onnxruntime/core/mlas/lib/aarch64/DepthwiseQConvKernelSize9Neon.S

```diff
@@ -0,0 +1,366 @@
+        stp     x25, x26, [sp, #.LMlasConvSymDepthwiseKernelSize9_backup_x25_x26]
+        stp     x27, x28, [sp, #.LMlasConvSymDepthwiseKernelSize9_backup_x27_x28]
+        stp     d8, d9, [sp, #.LMlasConvSymDepthwiseKernelSize9_backup_d8_d9]
+        stp     d10, d11, [sp, #.LMlasConvSymDepthwiseKernelSize9_backup_d10_d11]
+        stp     d12, d13, [sp, #.LMlasConvSymDepthwiseKernelSize9_backup_d12_d13]
+        stp     d14, d15, [sp, #.LMlasConvSymDepthwiseKernelSize9_backup_d14_d15]
+
+        mov     w10, #0x80808080
+        ldr     x9, [x5, #.LConvSymDepthwisePostProcessParams_Bias]
+        ldr     x8, [x5, #.LConvSymDepthwisePostProcessParams_Scale]
+        add     x5, x5, #.LConvSymDepthwisePostProcessParams_ZeroPoint
+        ins     v12.d[0], x1                // Filter
+        ins     v13.d[0], x9                // Bias
+        ins     v13.d[1], x8                // Scale
+        ld1r    {v0.8h}, [x5]               // zero point
+        dup     v5.4s, w10                  // flip 0x80
+
+        tbnz    x6, #.LMLAS_CONV_SYM_FLAG_PER_CHANNEL_SCALE_BIT_INDEX, .LMlasConvSymDepthwiseKernelSize9_SkipPerTensorScaleInit
+        ld1r    {v1.4s}, [x8]               // load scale value
+        mov     v2.16b, v1.16b
+        mov     v3.16b, v1.16b
+        mov     v4.16b, v1.16b
+
+.LMlasConvSymDepthwiseKernelSize9_SkipPerTensorScaleInit:
+
+        add     x9, x3, x2                  // x9 <---- Ouput1
+        cbz     x4, .LMlasConvSymDepthwiseKernelSize9_Exit
+
+.LMlasConvSymDepthwiseKernelSize9_OutputLoop:
+        ldp     x20, x21, [x0], #72         // input ptrs for Output0
+        ldp     x22, x23, [x0, #-56]
+        sub     x4, x4, #1
+        ldp     x24, x25, [x0, #-40]
+        ldp     x26, x27, [x0, #-24]
+        ldur    x28, [x0, #-8]
+        
+        cbz     x4, .LMlasConvSymDepthwiseKernelSize9_Dup_Inputs
+        ldp     x10, x11, [x0], #72         // input ptrs for Output0
+        ldp     x12, x13, [x0, #-56]
+        sub     x4, x4, #1
+        ldp     x14, x15, [x0, #-40]
+        ldp     x16, x17, [x0, #-24]
+        ldur    x19, [x0, #-8]
+        b       .LMlasConvSymDepthwiseKernelSize9_Loaded_Input
+
+.LMlasConvSymDepthwiseKernelSize9_Dup_Inputs:
+        mov     x9, x3                      // Output1 <-- Output0
+        mov     x10, x20
+        mov     x11, x21
+        mov     x12, x22
+        mov     x13, x23
+        mov     x14, x24
+        mov     x15, x25
+        mov     x16, x26
+        mov     x17, x27
+        mov     x19, x28
+
+.LMlasConvSymDepthwiseKernelSize9_Loaded_Input:
+
+        eor     x8, x8, x8                  // Processed channels
+        umov    x1, v12.D[0]                // filter
+        umov    x5, v13.D[0]                // bias
+        umov    x7, v13.D[1]                // scale
+    
+        cmp     x8, x2                      // Save one register by not using count down to zero here
```

Comment:
x8 is always 0 here?  if so, is this conditional jump necessary here? this cost two cycles

Reply:
inc later. here is one time.

Reply:
Anyway, it is same to handle zero len